roles/nfs_volumes/library/partitionpool.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/python
+"""
+Ansible module for partitioning.
```

Comment:
It's not generic partitioning module, it has very limited functionality. I doubt it could ever make it to Ansible core/extra modules.